mesh-worker-service/src/main/java/io/functionmesh/compute/util/FunctionsUtil.java

```diff
@@ -75,9 +77,10 @@
     public static V1alpha1Function createV1alpha1FunctionFromFunctionConfig(String kind, String group, String version
             , String functionName, String functionPkgUrl, FunctionConfig functionConfig
             , String cluster, MeshWorkerService worker) {
-        Map<String, Object> customConfigs = worker.getWorkerConfig().getFunctionsWorkerServiceCustomConfigs();
+        MeshWorkerServiceCustomConfig customConfig = worker.getMeshWorkerServiceCustomConfig();
         CustomRuntimeOptions customRuntimeOptions = CommonUtil.getCustomRuntimeOptions(functionConfig.getCustomRuntimeOptions());
         String clusterName = CommonUtil.getClusterName(cluster, customRuntimeOptions);
+        String serviceAccountName = customRuntimeOptions.getServiceAccountName();
```

Comment:
It's insecure to allow users to specify arbitrary `serviceAccountName` to run functions in API requests, otherwise the user may intentionally or accidentally use a privileged service account and cause security issues.

Reply:
I have added a flag to the config to check whether mesh worker service allows user to customize the service account name or not.